### PR TITLE
MUZIMA-822

### PR DESCRIPTION
**Pull request by samuelowino**

### CODE REVIEW

**Comment by samuelowino** [comment]
Hey @benamonya you can use this apk for testing: https://drive.google.com/file/d/1IuEgkjnS7ceg7Y0uy5v2GkfJ9jlr7S1s/view?usp=sharing

Let me know if it works

File: app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java [hunk 1]
```diff
@@ -30,10 +30,13 @@
 import com.muzima.R;
 import com.muzima.adapters.ListAdapter;
 import com.muzima.adapters.setupconfiguration.GuidedSetupActionLogAdapter;
+import com.muzima.api.model.Form;
```
Comment (mssavai) [inline app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java:33]: @samuelowino is this import necessary?

File: app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java [hunk 2]
```diff
@@ -30,10 +30,13 @@
...
 import com.muzima.api.model.Location;
 import com.muzima.api.model.SetupConfigurationTemplate;
+import com.muzima.controller.FormController;
 import com.muzima.controller.LocationController;
 import com.muzima.controller.SetupConfigurationController;
+import com.muzima.model.DownloadedForm;
```
Comment (mssavai) [inline app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java:39]: @samuelowino is this import necessary?

File: app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java [hunk 3]
```diff
@@ -46,16 +49,19 @@
 
 import net.minidev.json.JSONObject;
 
+import java.io.IOException;
```
Comment (mssavai) [inline app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java:52]: @samuelowino is this import necessary?

Comment (samuelowino) [reply app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java:52]: 😀 l forgot to remove them, let me resolve that...

File: app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java (hunk 1 above)
Comment (samuelowino) [reply app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java:33]: This particular one seems to be in use

LoC Ref: 372 - 378

**Pull request merged**